src/controllers/PostController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import { IPostRepository } from "../repositories/ContentfulRepository.ts";
+
+class PostController {
+  constructor(postRepository: IPostRepository) {
```

Comment:
Suggestion
I would inject something like IPostService, not the repository
For the moment, your IPostService will do nothing, just proxy data from repository

Reply:
For now i dont have IPostService at all
So I need to use IPostService and IPostRepository?

Reply:
Yep
Controller injects service, service injects repository